src/test/java/windycall/DeadlineTest.java

```diff
@@ -0,0 +1,25 @@
+package windycall;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void standardDateFormatTest() {
```

Comment:
I think the usual convention for testing method names should start with `should...` to say its expected behaviour, i.e. `shouldParseFormattedDate`. Do you want to follow that convention?

Reply:
I think the method name is fine.
According to the [Coding Standard](https://se-education.org/guides/conventions/java/intermediate.html), the test function can be featureUnderTest_testScenario_expectedBehavior(), where the testScenario and expectedBehaviour are optional.